Dockerfile

```
@@ -1,96 +1,75 @@
-        echo "Changing ubuntu user id to $PUID"
-        usermod -u "$PUID" ubuntu
-        gosu ubuntu /opt/subnet-2/.venv/bin/python3 "$@"
+        usermod -u "$PUID" subnet2
+        exec gosu subnet2 "$@"
     fi
 else
-    gosu ubuntu /opt/subnet-2/.venv/bin/python3 "$@"
+    exec gosu subnet2 "$@"
 fi
 EOF
 RUN chmod +x /entrypoint.sh
```

Hadolint (2.14.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)